### DISCUSSION TITLE
Clicks and pops in looping audio samples

### DISCUSSION THREAD

**Discussion by Pdz1** [FakeFishGames/Barotrauma#15361] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Clicking sounds occur during transitions between audio samples of active installations. This currently affects the ventilation, oxygen generator, and reactor (though less noticeably on the reactor).

https://github.com/user-attachments/assets/3c244033-0081-4a46-be1e-8528539e6529

https://github.com/user-attachments/assets/ce116674-89e0-41c3-853e-6bbe36305912

https://github.com/user-attachments/assets/3ff24400-1191-4a5e-bb25-d162c71dae01

### How I Fixed It
I was able to remove the clicks by editing the audio samples and applying zero-crossing alignment

https://github.com/user-attachments/assets/2f2e3a38-b651-49c7-9fbf-2842c214bf7e

https://github.com/user-attachments/assets/a416b156-163a-4988-9d46-421e8f988914

https://github.com/user-attachments/assets/8d7f5b62-b954-4c68-bcb0-93c6543e8e3d

### Reproduction steps

1. Select any submarine.
2. Position yourself near an active ventilation system, oxygen generator, or reactor.
3. Listen for clicking sounds during sample transitions.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Addressed in https://github.com/FakeFishGames/Barotrauma-development/commit/fb48795acc7cf1280bfc8639e8e587d7bf581ff1 (commit in our private repo). Fix coming up in a future unstable build.

**Comment by Pdz1** [comment]
v1.8.6.2. Still not fixed
